.ci/static-checks.sh

```diff
@@ -175,6 +175,52 @@ check_versions()
 	fi
 }
 
+# Ensure all files (where possible) contain an SPDX license header
+check_license_headers()
+{
+	# See: https://spdx.org/licenses/Apache-2.0.html
+	local -r spdx_tag="SPDX-License-Identifier"
+	local -r spdx_license="Apache-2.0"
+	local -r pattern="${spdx_tag}: ${spdx_license}"
+
+	echo "INFO: Checking for SPDX license headers"
+
+	# List of files to check for a license header inside
+	files=$(git diff-tree \
+		--name-only \
+		--no-commit-id \
+		--diff-filter="ACMRUX" \
+		-r \
+		origin/master HEAD || true)
+
+	# no files were changed
+	[ -z "$files" ] && echo "INFO: No files found" && return
+
+	missing=$(egrep \
+		--exclude="*.json" \
+		--exclude=".git/*" \
+		--exclude=".gitignore" \
+		--exclude="Gopkg.lock" \
+		--exclude="LICENSE" \
+		--exclude="protocols/grpc/*.pb.go" \
+		--exclude="vendor/*" \
+		--exclude="VERSION" \
+		--exclude="*.md" \
+		-EL "\<${pattern}\>" \
```

Comment:
This list of excludes works reasonably well, but I wonder if we want to add `.pullapprove.yml` to it too? We could of course instead add the SPDX header to that file ;)

Reply:
what about exclude yml and toml files in general?

Reply:
I guess we could, although for example the versions database does have the SPDX tag:

- https://github.com/kata-containers/runtime/blob/master/versions.yaml#L4

But yes, we'd need to add it to `configuration.toml` (and probably should ;)

Reply:
I vote maybe we should add them if that does not upset any of the relevant processing tools or bots

Reply:
Ok - added.